### PR TITLE
Fix Sentry issue

### PR DESCRIPTION
**Pull request by Phyks**
https://sentry.io/organizations/dissemin/issues/894371243/?project=189330&query=is%3Aunresolved&statsPeriod=14d&utc=true.

Should be fixed I think, according to https://github.com/dissemin/dissemin/blob/master/papers/urls.py#L33-L34.

### CODE REVIEW

File: papers/views.py [hunk 1]
```diff
@@ -328,7 +328,7 @@ def refetchResearcher(request, pk):
     from backend.tasks import fetch_everything_for_researcher
     fetch_everything_for_researcher.delay(pk=pk)
 
-    return redirect(reverse('researcher', researcher=pk))
+    return redirect(reverse('researcher', slug=researcher.slug))
```
Comment (wetneb) [inline papers/views.py]: I think the pk should be required too - we cannot identify a researcher by its slug only. I assume that this part of the code is not tested yet.

Comment (Phyks) [reply papers/views.py]: Indeed, I'm rebuilding the VM at the moment. We need both researcher and slug I think.

Comment (wetneb) [reply papers/views.py]: and we need it to be wrapped in an `args={...}` keyword too (hence the error)

Comment (wetneb) [reply papers/views.py]: anyway it would be good to confirm that with a unit test (I can add it)

Comment (Phyks) [reply papers/views.py]: Tested in my VM and fixed. There was actually already a test, except it is only checking the case when the researcher is not logged in. Not sure how to fix this test :/ https://github.com/dissemin/dissemin/pull/620/files#diff-2e713353c3d10b8fe4b78e8759e15dc0R116

**Comment by Phyks** [comment]
Thanks for the unittest!

**Pull request merged**